src/pages/_proxied-dot-io/packer/plugins/[...page].tsx

```diff
@@ -56,6 +56,7 @@ function DocsView({ isDevMissingRemotePlugins, ...props }) {
         additionalComponents={additionalComponents}
         baseRoute={basePath}
         product={product}
+        // eslint-disable-next-line @typescript-eslint/ban-ts-comment
```

Comment:
I didn't love disabling it but I had no idea what to put as the "reason" it was requiring other than "because @dstaley  but it here" 🤣

Reply:
I'm gonna go ahead and figure out why this `ts-expect-error` is even needed and make whatever changes necessary to get rid of it 💪

Reply:
Love that! Thanks @dstaley 😊

Reply:
Also, wasn't intending for this to be a call-out. 🤣 I just tagged you because git-blame said you did it here in `dev-portal`. I assumed it was moved over from a different repo!